src/lib/log/log.c

```diff
@@ -834,6 +834,23 @@ logs_free_all(void)
    * that happened between here and the end of execution. */
 }
 
+/** Close signal-safe log files. */
```

Comment:
Could we have a little more documentation about when this function is appropriate to call?

Reply:
Note to self: this function can be called from a signal handler. It should only be called when shutting down the log module. It's designed to be called by the err module, when terminating the process on an abnormal condition.

Reply:
Fixed in 671ec51163ec69e8b632bb121ae60354b9bd5b1d